src/System.Runtime.InteropServices.RuntimeInformation/src/PinvokeAnalyzer_win8-wpa81-APIs.txt

```diff
@@ -0,0 +1 @@
+api-ms-win-core-sysinfo-l1-2-0.dll!GetNativeSystemInfo
```

Comment:
I've validated that this is supported by running WACK on all apps.  We need to get buildtools updated to handle more fxs in p-invoke checker.